Cakes-Co/css/style.css

```diff
@@ -0,0 +1,312 @@
+}
+
+.logo {
+  max-width: 100px;
+}
+
+.branding {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.branding p {
+  font-family: var(--heading-font-family);
+  font-size: 1.4rem;
+  /* margin-top: -20px; */
+  /* text-shadow: 0 -1px var(--gold); */
+  color: var(--gold);
+}
+
+#menu {
+  display: none;
+  text-align: center;
+}
+
+.menu_item {
+  display: block;
+  background-color: var(--light-pink);
+  padding: 1rem 2rem;
+}
+.menu_item:hover {
+  background-color: var(--darker-pink);
+  color: var(--white);
+}
+
+.icon {
+  display: block;
+  position: absolute;
+  right: 10px;
+  top: 10px;
+  font-size: 32px;
+}
+
+/* medium */
+@media (min-width: 768px) {
+  #menu {
+    display: flex!important;
+    justify-content: center;
+    gap: 1rem;
+    margin-top: 0.2rem;
+    background-color: var(--light-pink);
+  }
+  .icon {
+    display: none;
+  }
+}
+
+/* large */
+@media (min-width: 1024px) {
+  .header {
+    height: 150px;
+    padding: 2rem 3rem;
+  }
+
+  .logo {
+    max-width: 170px;
+  }
+  nav {
+    display: flex;
+    justify-content: center;
+  }
+
+  #menu {
+    width: 90%;
+    justify-content: right;
+    padding: 1rem;
+    gap: 0.5rem;
+    margin: 1.5rem 4rem;
+    flex-grow: 1;
+  }
+  .menu_item {
+    color: var(--white);
+    background-color: var(--darker-pink);
+  }
+  .menu_item:hover {
+    background-color: var(--light-pink);
+    color: var(--gold);
+  }
+}
+
+/* Main Styles */
+
+main {
+  padding: 1.25rem;
+}
+
+.welcome {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+
+.cupcake-group {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  gap: 1rem;
+  margin-bottom: 2rem;
+}
+
+.cupcake-container {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.middle-btn {
+  transition: 0.5s ease;
+  opacity: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+}
+
+.cupcake_item-img {
+  transition: 0.5s ease;
+  opacity: 1;
+  width: 100%;
+}
+
+.cupcake-container:hover .cupcake_item-img {
+  opacity: 0.3;
+}
+
+.cupcake-container:hover .middle-btn {
+  opacity: 1;
+}
+
+#hero-img {
+  height: 100%;
+  max-width: 100%;
+  object-fit: cover;
+  order: 1;
+}
+
+.welcome-text {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  text-align: justify;
+  margin: 1rem 0;
+}
+/* medium */
+@media (min-width: 768px) {
+  .welcome {
+    flex-direction: row-reverse;
+    gap: 2rem;
+    align-items: center;
+  }
+  .welcome-text h1 {
+    text-align: right;
+  }
+  #hero-img {
+    max-width: 50%;
+  }
+}
+
+/* large */
+@media (min-width: 1024px) {
+  main {
+    margin: 0 4rem;
+    padding: 0;
+  }
+  #hero-img {
+    max-width: 40%;
+  }
+
+  p {
+    font-size: 1.2rem;
+  }
+
+  h1 {
+    font-size: 2.2rem;
+  }
+
+  h3 {
+    font-size: 1.8rem;
+  }
+  .cupcake-group {
```

Comment:
I really like how seemlessly the cupcakes move from the mobile to the desktop view :)